packages/p2p/src/components/my-ads/ad-type.scss

```diff
@@ -1,15 +1,19 @@
-.ad-type {
+.ad-types {
```

Comment:
had to change the className as it wasn't rendering into the row. `display: flex` kept on getting overridden by another class when using ad-type className